### ISSUE TITLE
Somes links are not well displayed in the Back Office sidebar

### ISSUE THREAD

**Issue by PrestaEdit** [PrestaShop/PrestaShop#27567]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

Somes links, as the "Marketing" tab is not well displayed on the legacy controller instead of modern ones.

<img width="215" alt="Capture d’écran 2022-02-02 à 15 04 37" src="https://user-images.githubusercontent.com/2631425/152168949-cbb9147b-e1db-41f8-86ff-66c8d6857ab9.png">
<img width="215" alt="Capture d’écran 2022-02-02 à 15 04 53" src="https://user-images.githubusercontent.com/2631425/152168954-f1e8b695-dd07-4d24-99ca-0985ec41078a.png">

### Expected behavior

Good display.

### Steps to reproduce

1. Go to a modern page and see the link.
2. Go to a legacy page and see the display.

### PrestaShop version(s) where the bug happened

1.7.8.3

**Comment by hibatallahAouadni** [comment]
Hello @PrestaEdit 

You just need to reset your Facebook module :sweat_smile: 
Closing the issue as it's not even a bug, it's just cache of the module.

Thanks!

**Comment by PrestaEdit** [comment]
Actually... the Facebook module is uninstall.

**Comment by hibatallahAouadni** [comment]
>Actually... the Facebook module is uninstall.

So can you check `Marketing with Google` module?
Waiting for your feedback.

Thanks!

**Comment by PrestaEdit** [comment]
The two of them. I delete them every time I install a new version in order to tests.

It's seems that the "Marketing" tab is added without relying on a module and as you say, maybe the fact that the Marketing tab is not well handled is due to a specific CSS rule into one of this module and as the tab is not mention to be inside a module, it not removed. No worries, it's more PrestaShopCorp issue than PrestaShop, here.

**Comment by hibatallahAouadni** [comment]
Hello @PrestaEdit 

I didn't reproduce the issue with PS1783 after uninstall both modules `ps_facebook` and `Marketing with Google`, see the attached screen record below:

https://user-images.githubusercontent.com/23308427/152202880-af5a9c4b-b133-450c-8bd5-902c587e92d4.mp4

Waiting for your feedback.
Thanks!

**Comment by lephuongbg** [comment]
@hibatallahAouadni Freshly installed Prestashop 1.7.8.3 still has that problem.

Uninstall Facebook and Google modules **the first time** does not make the "Marketing" menu go away, and it still looks broken. 

But when I re-installed Facebook module, the menu became fixed and looked "correct" like in your video. Then I uninstalled Facebook module **for the second time**, only then the "Marketing" menu went away.

tldr; uninstall -> reinstall -> uninstall Facebook module will fix the menu.
